content/documentation/getting-started/index.adoc

```diff
@@ -318,13 +339,15 @@ deployment:
 
 icon:lightbulb[size=1x]{nbsp} If `accessible_namespaces` has an entry with the special value of `+++**+++` (two asterisks), it denotes that Kiali be given access to all namespaces via a single cluster role (if using this special value of `+++**+++`, you are required to have already granted the operator permissions to create cluster roles and cluster role bindings). It is not recommended for production but the following command will create both the Kiali operator and the Kiali CR, configured for full cluster access to current and new namespaces:
 
-icon:bullhorn[size=1x]{nbsp} If the operator was originally installed with --accessible_namespaces not set to `+++**+++`, you cannot later edit the Kiali CR and change accessible_namespaces to `+++**+++`. You must use the bash command below to reinstall the operator so that it can be granted the additional permissions required.
+icon:bullhorn[size=1x]{nbsp} If the operator was not originally installed with --accessible_namespaces set to `+++**+++`, you cannot later edit the Kiali CR and change accessible_namespaces to `+++**+++`. You must use the bash command below to reinstall the operator so that it can be granted the additional permissions required.
 
 [source,bash]
 ----
-bash <(curl -L https://git.io/getLatestKialiOperator) -an '**'
+  bash <(curl -L https://git.io/getLatestKialiOperator) -an '**'
 ----
 
+Maistra supports multi-tenancy and the `accessible_namespaces` extends that feature to Kiali. However, explicit naming of accessible namespaces can benefit non-Maistra installations as well, with it Kiali does not need cluster roles and the Kiali Operator does not need super-create-cluster-roles-permission.
```

Comment:
lol... I don't think we want to use my "super-create-cluster-roles-permission" term - that was my  "I don't know what to call it, but its very powerful". Perhaps say:

`does not need permissions to create cluster roles`

Reply:
such a stickler